### ISSUE TITLE
PONCHO: Analyze Uses Old Format?

### ISSUE THREAD

**Issue by dthain** [cooperative-computing-lab/cctools#2651]
Using the current master, `poncho_package_analyze` gives the following output:

```
%  ./poncho_package_analyze example.py package.json
% cat package.json 
{
    "channels": [
        "conda-forge",
        "defaults"
    ],
    "dependencies": [
        "matplotlib=3.3.4=py38h06a4308_0",
        "pip=20.2.4=py38h06a4308_0",
        "python=3.8.5=h7579374_1"
    ],
    "name": "base",
    "prefix": "/afs/crc.nd.edu/user/d/dthain/miniconda3"
}
```

But this isn't the format expected by `poncho_package_create`.
https://cctools.readthedocs.io/en/latest/poncho/

Is something out of sync?

**Comment by dthain** [comment]
Per our discussion just now, Barry is going to update the new format to embed the old format in the `conda` section.  (And also update the spec

**Comment by dthain** [comment]
fixed by #2655
